pkg/resourceinterpreter/defaultinterpreter/aggregatestatus.go

```diff
@@ -264,3 +265,50 @@ func aggregateStatefulSetStatus(object *unstructured.Unstructured, aggregatedSta
 
 	return helper.ToUnstructured(statefulSet)
 }
+
+func aggregatePodStatus(object *unstructured.Unstructured, aggregatedStatusItems []workv1alpha2.AggregatedStatusItem) (*unstructured.Unstructured, error) {
+	pod, err := helper.ConvertToPod(object)
+	if err != nil {
+		return nil, err
+	}
+
+	newStatus := &corev1.PodStatus{}
+	newStatus.ContainerStatuses = make([]corev1.ContainerStatus, 0)
+	readySum := 0
+	containerSum := 0
+	for _, item := range aggregatedStatusItems {
+		if item.Status == nil {
+			continue
+		}
+
+		temp := &corev1.PodStatus{}
+		if err = json.Unmarshal(item.Status.Raw, temp); err != nil {
+			return nil, err
+		}
+
+		for _, containerStatus := range temp.ContainerStatuses {
+			tempStatus := containerStatus
+			newStatus.ContainerStatuses = append(newStatus.ContainerStatuses, tempStatus)
```

Comment:
IMOP, the status of the collection container does not appear to be required.

Reply:
> IMOP, the status of the collection container does not appear to be required.

I think it will show more information. If it is not needed, I can remove it.

Reply:
> IMOP, the status of the collection container does not appear to be required.

The status of containers is needed to be collected. Or the column of READY will always be 0/N.

Reply:
> The status of containers is needed to be collected. Or the column of READY will always be 0/1.

I'm not sure about it. I thought the `Ready` status is based on the `.status.phase`. @lonelyCZ might know it.

Reply:
> 

I've already tried. ready status is based on the status of the container.

Reply:
> > The status of containers is needed to be collected. Or the column of READY will always be 0/1.
> 
> I'm not sure about it. I thought the `Ready` status is based on the `.status.phase`. @lonelyCZ might know it.

I've already tried. ready status is based on the status of the container.

Reply:
> Hi @xyz2277, how about only collecting the fields we need.

I tried too. When you see pod details, a lot of empty field will be found. Is that ok?

Reply:
> Hi @xyz2277, how about only collecting the fields we need.

I tried too. When you see pod details, a lot of empty field will be found.

Reply:
It will be ok.